Build instructions don't work

**Describe the bug**

When building liquidsoap from source I get this error:

```
File "src/runtime/dune", line 4, characters 7-17:
4 |   (pps ppx_string))
           ^^^^^^^^^^
Error: Library "ppx_string" not found.
-> required by library "liquidsoap_runtime" in _build/default/src/runtime
-> required by executable liquidsoap in src/bin/dune:2
-> required by _build/default/src/bin/liquidsoap.exe
-> required by _build/default/liquidsoap.config
-> required by alias install
-> required by alias default in dune:20
File "src/lang/dune", line 84, characters 55-64:
84 |  (libraries liquidsoap-lang.console dune-site str unix menhirLib)
                                                            ^^^^^^^^^
Error: Library "menhirLib" not found.
-> required by library "liquidsoap-lang" in _build/default/src/lang
-> required by _build/default/META.liquidsoap-lang
-> required by _build/install/default/lib/liquidsoap-lang/META
-> required by _build/default/liquidsoap-lang.install
-> required by alias install
-> required by alias default in dune:20
File "src/lang/dune", line 83, characters 7-17:  
83 |   (pps sedlex.ppx ppx_string))
            ^^^^^^^^^^
Error: Library "sedlex.ppx" not found.
-> required by _build/default/src/lang/json_parser__mock.ml.pp.mock
-> required by _build/default/src/lang/json_parser__mock.mli.inferred
-> required by _build/default/src/lang/json_parser.ml
-> required by _build/install/default/lib/liquidsoap-lang/json_parser.ml
-> required by _build/default/liquidsoap-lang.install
-> required by alias install
-> required by alias default in dune:20
```

**To Reproduce**

```
git clone https://github.com/savonet/liquidsoap
dune build
```

**Expected behavior**

Liquidsoap should build based on the README instructions and documentation.

**Version details**

- OS: Arch Linux
- Version: rolling-release-v2.2.x-1-g453b2c0f7

**Install method**

I installed opam from my distro and dune from opam

This still fails:

```<><> Synchronising pinned packages ><><><><><><><><><><><><><><><><><><><><><><>
[NOTE] Ignoring uncommitted changes in /home/jookia/liquidsoap (`--working-dir' not active).
[liquidsoap.2.2.0] synchronised (no changes)

[ERROR] Package conflict!
  * Missing dependency:
    - dtools >= 0.4.5
    no matching version

No solution found, exiting```

dtools, cry and metadata have invalid versions that don't exist

--- a/README.md
+++ b/README.md
@@ -96,6 +96,10 @@ To build the code, you will need all the required `opam` packages installed. You
 
 ```sh
 cd liquidsoap/
+opam pin git+https://github.com/savonet/ocaml-dtools
+opam pin git+https://github.com/savonet/ocaml-cry
+opam pin git+https://github.com/savonet/ocaml-metadata
+opam pin git+https://github.com/savonet/ocaml-ffmpeg # optional
 opam pin -ny .
 opam install --deps-only liquidsoap
 ```

